### PR TITLE
es6-classes by jaden bertino

### PR DESCRIPTION
**Pull request by jadenbertino**
image
image

### CODE REVIEW

File: es6-classes/square.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+
+class Square extends Shape {
+  constructor(area, circumference, width) {
```
Comment (scrumptiousdev) [inline es6-classes/square.js]: Per instruction:
Write its constructor to take the **square's width as a parameter** (and nothing else)

File: es6-classes/circle.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+
+class Circle extends Shape {
+  constructor(area, circumference, radius) {
```
Comment (scrumptiousdev) [inline es6-classes/circle.js]: Per instruction:
Write its constructor to take the **circle's radius as a parameter** (and nothing else)

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]

**Comment by jadenbertino** [comment]
When I first did the exercise I was a confused by the instructions because if the Square and Circle classes each accept one argument then the `.print()` method looks kind of strange on each:

image
image

With that being said each class only accepts one argument now 👍

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]
Changes requested via Slack msg :)
This is just to keep the recording

**Comment by jadenbertino** [comment]
### The `Square` class now calculates area and circumference based on the `width` parameter:
image

### The `Circle` class now calculates area and circumference based on the `radius` parameter:
image

**Review by scrumptiousdev — APPROVED** [review]
Approved! :)

**Pull request merged**